Sources/Profile-Modify.php

```diff
@@ -2296,7 +2296,7 @@ function alert_count($memID, $unread = false)
 	$smcFunc['db_free_result']($request);
 
 	// If this isn't the current user, get their boards.
-	if (isset($user_info) && $user_info['id'] != $memID)
+	if (!isset($user_info) || $user_info['id'] != $memID)
```

Comment:
you access $user_info without knowing if it exists,
this case could happen when this is called by the cronjob i think.

Reply:
hmm because of the "or" php stop the evaluation of the right side because the left side is already true...

Reply:
Exactly. :)